api/src/services/organizations_v1/remove_user_from_organization.py

```diff
@@ -0,0 +1,126 @@
+    )
+
+    admin_user_ids = set(db_session.execute(admin_user_ids_stmt).scalars().all())
+
+    # Check if target user is an admin and if they're the only admin
+    if target_user_id in admin_user_ids and len(admin_user_ids) == 1:
+        raise_flask_error(403, message="Cannot remove the last administrator from organization")
+
+
+def validate_organization_user_exists(
+    db_session: db.Session, user_id: UUID, organization: Organization
+) -> OrganizationUser:
+    """Validate that the user is a member of the specified organization.
+
+    Args:
+        db_session: Database session
+        user_id: ID of user to validate
+        organization: Organization to check membership in
+
+    Returns:
+        OrganizationUser: The organization user record
+
+    Raises:
+        FlaskError: 404 if user is not a member of the organization
+    """
+    org_user = db_session.execute(
+        select(OrganizationUser)
+        .where(OrganizationUser.organization_id == organization.organization_id)
+        .where(OrganizationUser.user_id == user_id)
+    ).scalar_one_or_none()
+
+    if not org_user:
+        raise_flask_error(404, message=f"Could not find User with ID {user_id}")
+
+    return org_user
+
+
+def remove_user_from_organization(
+    db_session: db.Session, user: User, target_user_id: UUID, organization_id: UUID
+) -> None:
+    """Remove a user from an organization, after validating permissions and business rules.
+
+    Args:
+        db_session: Database session
+        user: User making the request
+        target_user_id: ID of user to remove from organization
+        organization_id: ID of organization to remove user from
+
+    Raises:
+        FlaskError: 403 if forbidden, 404 if not found
+    """
+    logger.info(
+        "Attempting to remove user from organization",
+        extra={
+            "requesting_user_id": user.user_id,
+            "target_user_id": target_user_id,
+            "organization_id": organization_id,
+        },
+    )
+
+    # Get organization and validate it exists
+    organization = get_organization(db_session, organization_id)
+
+    # Check if requesting user has permission to manage org members
+    if not can_access(user, {Privilege.MANAGE_ORG_MEMBERS}, organization):
+        raise_flask_error(403, "Forbidden")
+
+    # Validate target user is a member of the organization
+    org_user = validate_organization_user_exists(db_session, target_user_id, organization)
+
+    # Check last admin protection
+    check_last_admin_protection(db_session, target_user_id, organization)
+
+    # Perform the deletion - cascade delete will handle OrganizationUserRole records
+    db_session.delete(org_user)
+
+    logger.info(
+        "Successfully removed user from organization",
+        extra={
+            "requesting_user_id": user.user_id,
+            "target_user_id": target_user_id,
+            "organization_id": organization_id,
+        },
+    )
```

Comment:
Nitpicky, but we can exclude the extra here. You added `target_user_id` and `organization_id` at the start of the request. The `requesting_user_id` sort've already exists as we put the auth user ID automatically on requests as well.

Same with the log message earlier.